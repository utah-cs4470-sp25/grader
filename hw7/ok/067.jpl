fn f(x: void) : int {
   return 1
}
show f(void)
