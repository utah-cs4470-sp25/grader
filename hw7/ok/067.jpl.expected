(FnCmd f (((VarLValue x) (VoidType))) (IntType) (ReturnStmt (IntExpr (IntType) 1)))
(ShowCmd (CallExpr (IntType) f (VoidExpr (VoidType))))
